crates/provider-wit-bindgen/src/lib.rs

```diff
@@ -94,13 +93,16 @@ struct ProviderBindgenConfig {
     pub(crate) exposed_interface_deny_list: Vec<LatticeExposedInterface>,
 
     /// wit-bindgen configuration that is passed straight through (uses vendored wit-bindgen)
-    pub(crate) wit_bindgen_cfg: WitBindgenConfig,
+    pub(crate) wit_bindgen_cfg: Option<WitBindgenConfig>,
 
     /// Translation strategy for functions on imported interfaces when generating lattice data structures and code
     pub(crate) import_fn_lattice_translation_strategy: WitFunctionLatticeTranslationStrategy,
 
     /// Translation strategy for functions on exported interfaces when generating lattice data structures and code
     pub(crate) export_fn_lattice_translation_strategy: WitFunctionLatticeTranslationStrategy,
+
+    /// Whether to replace WIT-ified maps (`list<tuple<T, T>>`) with a Map type (`std::collections::HashMap`)
+    pub(crate) replace_witified_maps: Option<bool>,
```

Comment:
Why `Option` instead of just a bool?

Reply:
This can be a bool -- will update

- [ ] change `replace_witified_maps` to a bool